ios_widget_catalog_compare/ios/.gitignore

```
@@ -30,3 +30,6 @@ Runner/GeneratedPluginRegistrant.*
 !default.mode2v3
 !default.pbxuser
 !default.perspectivev3
+
+# misc
+build/
```

Restore the end newline.
```suggestion
build/

```